shinysdr/test/plugins/wspr/test_demodulator.py

```diff
@@ -86,6 +87,12 @@ def output_message(self, message):
         self.messages.append(message)
 
 
+STATUS_IDLE = 'Waiting for next even minute'
+STATUS_RECEIVING = 'Receiving'
+STATUS_DECODING = 'Decoding…'
+STATUS_DECODING_AND_RECEIVING = STATUS_DECODING
```

Comment:
If you're going to define constants for these things, maybe they should be in demodulator.py?